python/python/glide/async_commands/transaction.py

```diff
@@ -949,6 +949,67 @@ def linsert(
             RequestType.LInsert, [key, position.value, pivot, element]
         )
 
+    def lmove(
+        self: TTransaction,
+        source: str,
+        destination: str,
+        wherefrom: ListDirection,
+        whereto: ListDirection,
+    ) -> TTransaction:
+        """
+        Atomically pops and removes the left/right-most element to the list stored at `source`
+        depending on `wherefrom`, and pushes the element at the first/last element of the list
+        stored at `destination` depending on `whereto`.
+
+        Notes:
+            When in cluster mode, both `source` and `destination` must map to the same hash slot.
```

Comment:
We don't add this to transction.
It is applicable to the entire transaction, rather than to a particular command(s) in it.

Please Add since section